tracing-flame/Cargo.toml

```diff
@@ -1,8 +1,26 @@
 [package]
 name = "tracing-flame"
 version = "0.1.0"
-authors = ["Jane Lusby <jlusby@yaah.dev>"]
+authors = [
+    "Jane Lusby <jlusby@yaah.dev>",
+    "Eliza Weisman <eliza@buoyant.io>",
+    "David Barsky <me@davidbarsky.com>",
```

Comment:
take it or leave it: i don't personally feel the need to be listed as an author for this crate...you did all the work :)

Reply:
Same here; I just reviewed it :)